sentry-android-replay/src/main/java/io/sentry/android/replay/SessionReplayOptions.kt

```diff
@@ -0,0 +1,31 @@
+package io.sentry.android.replay
+
+import io.sentry.SentryReplayOptions
+
+// since we don't have getters for redactAllText and redactAllImages, they won't be accessible as
+// properties in Kotlin, therefore we create these extensions where a getter is dummy, but a setter
+// delegates to the corresponding method in SentryReplayOptions
+
+/**
+ * Redact all text content. Draws a rectangle of text bounds with text color on top. By default
+ * only views extending TextView are redacted.
+ *
+ * <p>Default is enabled.
+ */
+var SentryReplayOptions.redactAllText: Boolean
+    @Deprecated("Getter is unsupported.", level = DeprecationLevel.ERROR)
+    get() = error("Getter not supported")
```

Comment:
l: Why is this deprecated?

Reply:
I explained in a comment on top of the file, but basically we scream to the dev that this should not be used to get the value. This extension only exists to set the value conveniently in Kotlin (`redactAllText = true` instead of `setRedactAllText(true)`), so only the setter matters here

Reply:
Thanks!! I didn't get the comment.

Can you just remove `get()` as in C# where you can have a property with just a setter?

Reply:
nope, unfortunately not possible in Kotlin. It's possible to only have a getter without a setter, but not the other way around